react-ui/src/components/Dashboard/Dashboard.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component, Fragment, Router } from "react";
```

Comment:
Please note that I moved Dashboard to its own folder as it is different from the typical Layouts and is used across all components (after login).

Reply:
That makes sense!